### DISCUSSION TITLE
The client on phone

### DISCUSSION THREAD

**Discussion by claubleh** [timschneeb/GalaxyBudsClient#534] [General]
Hello, how do I set up the client on my phone?
I noticed theres the app on google play but I dont have the money and I thought maybe I can set it up from the code but I dont know how.I know its not right but is there someone that can help me with setting up the client on my phone?

**Comment by timschneeb** [comment]
Hi, there's also a free version of the app available here: https://timschneeberger.me/downloads/me.timschneeberger.galaxybudsclient.demo_v5.1.0.1-Signed.apk (starting from the next release onwards, I'll properly attach these APKs on the release page in this repo)

(I posted about that originally in this issue: https://github.com/timschneeb/GalaxyBudsClient/issues/445#issuecomment-2251076994)

You can also manually compile from source code, but that isn't easy. To quickly sum it up, you would need to set up the .NET toolkit on a PC and follow these instructions: https://docs.avaloniaui.net/docs/guides/platforms/android/setting-up-your-developer-environment-for-android. Then clone this repo and use the [build script](https://github.com/timschneeb/GalaxyBudsClient/blob/master/GalaxyBudsClient.Android/build.sh) to compile the app.

**Reply by claubleh** [reply]
Thank you so much, but I must say, the main reason why I wanted to use this third party app was because my earbuds could not connect to the galaxy wearable app and it seems like this is also the case on this client.
In the app it says " disconnected" and it says that my earbuds are either out of range or in use by the official manager app, if I connect the earbuds only to bluetooth they work and I can lisen to music and they are not out of range, I think the problem might be with the earbuds but thank you for helping me expand my options and learn of this problem.
